fontir/src/ir.rs

```diff
@@ -32,6 +33,9 @@ pub struct StaticMetadata {
     /// See <https://learn.microsoft.com/en-us/typography/opentype/spec/head>.
     pub units_per_em: u16,
 
+    /// See <https://learn.microsoft.com/en-us/typography/opentype/spec/os2#fsselection>
+    pub selection_flags: SelectionFlags,
```

Comment:
I think the `fsSelection` name expands to "font _style_ selection" , so maybe it would read better if you name it `style_selection_flags` ("font" is implied of course); just `selection_flags` was unclear to me until I looked at the comment

Reply:
I well I see write_fonts calls them simply SelectionFlags, nevermind